### PR TITLE
ui - cut timeline length with last element on issue view

### LINKED ISSUE #7304: Locked issue UI broken
image

### PR DESCRIPTION
**Pull request by Cherrg**
cut timeline length with last element on issue view

fix #7304

- lightly enlight dark theme issue timeline color

(pls ignore whitespace on review)

Signed-off-by: Michael Gnehr <michael@gnehr.de>

## Image
* remove

grafik

### CODE REVIEW

File: templates/repo/issue/view_content.tmpl [hunk 1]
```diff
@@ -9,109 +9,69 @@
...
-							</div>
-						{{end}}
-					</div>
-					<div class="ui attached segment">
-						<div class="render-content markdown has-emoji">
-							{{if .Issue.RenderedContent}}
-								{{.Issue.RenderedContent|Str2html}}
-							{{else}}
-								<span class="no-content">{{.i18n.Tr "repo.issues.no_content"}}</span>
+			<div class="timeline-line">
```
Comment (silverwind) [inline templates/repo/issue/view_content.tmpl]: Instead of adding another wrapper div, would it instead be possible to add the class to the parent node?

Comment (Cherrg) [reply templates/repo/issue/view_content.tmpl]: Sometimes just more sleep helps.

Comment (Cherrg) [reply templates/repo/issue/view_content.tmpl]: yeah, found a way ... was just too easy ;)
changed now

**Review by mrsdizzie — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by zeripath** [comment]
Make lg-tm work

**Pull request merged**